infra/SECURITY.md

```
@@ -0,0 +1,139 @@
+
+### Verification
+
+To verify secrets are not in your image:
+
+```bash
+# Check image layers
+docker history your-image-name
+
+# Check for files in image
+docker run --rm your-image-name find / -name "*.env*" 2>/dev/null
+``` 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)